configure.ac

```diff
@@ -523,6 +523,7 @@ if test "$WITH_ECONF" = "yes" ; then
   PKG_CHECK_MODULES([ECONF], [libeconf], [],
   [AC_CHECK_LIB([econf],[econf_readDirs],[ECONF_LIBS="-leconf"],[ECONF_LIBS=""])])
   if test -n "$ECONF_LIBS" ; then
+    AC_CHECK_LIB([econf],[econf_errLocation], [], [AC_MSG_ERROR([Please update libeconf])])
```

Comment:
This configure snippet requires that `libeconf` either doesn't exist or is fresh enough, which is somewhat unusual.
I suppose it would be surprising for many to see the build fail by default in case` libeconf` is installed but not fresh enough.

Reply:
I have changed it to check the version of libeconf only.